src/client/src/consumer.rs

```diff
@@ -76,15 +76,12 @@ impl PartitionConsumer {
     ///
     /// ```no_run
     /// # use fluvio::{PartitionConsumer, Offset, ConsumerConfig, FluvioError};
-    /// # async fn do_fetch(consumer: &PartitionConsumer) -> Result<(), FluvioError> {
+    /// # async fn example(consumer: &PartitionConsumer) -> Result<(), FluvioError> {
     /// let response = consumer.fetch(Offset::beginning()).await?;
     /// for batch in response.records.batches {
     ///     for record in batch.records {
-    ///         if let Some(record) = record.value.inner_value() {
-    ///             let string = String::from_utf8(record)
-    ///                 .expect("record should be a string");
-    ///             println!("Got record: {}", string);
-    ///         }
+    ///         let string = String::from_utf8_lossy(record.value.as_ref());
```

Comment:
there is new `to_string_lossy` in the record

Reply:
Hmm that's right... I wish I had seen that before we added it, I think it was not a good idea to add `to_string_lossy` to our record type. Is this a blocker in your eyes or could we merge it like this anyways?

Reply:
@simlay opinion?

Reply:
I'm fine with it as is and we can remove to_string_lossy